.licensure.yml

```diff
@@ -0,0 +1,57 @@
+change_in_place: true
+excludes:
+  - Cargo.toml
+  - Cargo.lock
+  - LICENSE
+  - pgx-utils/assets/ansi.tmTheme
+  - ".envrc"
+  - ".gitignore"
+  - "flake\\..*"
+  - "logo.*"
+  - ".github/**/*"
+  - ".cargo/config"
+  - "cargo-pgx/src/templates/*"
+  - ".*\\.control"
+  - ".*\\.md"
+  - ".*\\.nix"
+  - ".*\\.yml"
+licenses:
+  - files: any
+    ident: PostgreSQL
+    authors:
+      - name: ZomboDB, LLC and Technology Concepts & Design, Inc.
+        email: support@tcdi.com
+    template: |
+      Portions Copyright 2020-2021 ZomboDB, LLC. 
+      Portions Copyright 2021-[year] Technology Concepts & Design, Inc. <support@tcdi.com>.
+      All rights reserved. 
```

Comment:
This term applied per-source-file seems to differ from the license in the root of the repo, creating an impression that these files follow a more restrictive license ("All rights reserved"). It should be updated in the root license, if this is included at all.